setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
```

Comment:
I believe there are `maintainer` and `maintainer_email` fields that you can use instead, which may be more appropriate given there isn't one person who wrote this. I'm not 100% on that, check the docs.

Also, maintainer (replacing author here) should probably include something human readable, like MOC haas developers or such.